person.rb

```diff
@@ -5,14 +5,15 @@
 
 class Person < Nameable
   attr_reader :id
-  attr_accessor :name, :age
+  attr_accessor :name, :age, :rentals
 
   def initialize(age, name = 'Unknown', parent_permission: true)
     super()
     @id = rand(1..1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
 
   def correct_name
```

Comment:
- [ ] As person has many **rentals**, kindly ensure that a method for adding rentals is present present in your `Pesron` class. The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association. It should look similar to this:
```ruby
  class Person < Nameable
  .
  .
  . 
  def add_rental(book, date)
    Rental.new(date, book, self)
  end
  .
  .
end
```